scarb/src/bin/scarb/args.rs

```diff
@@ -483,7 +483,7 @@ pub struct PackageArgs {
 #[derive(Parser, Clone, Debug)]
 pub struct PublishArgs {
     /// Registry index URL to upload the package to.
-    #[arg(long, value_name = "URL")]
+    #[arg(long, value_name = "URL", default_value = "https://scarbs.xyz/")]
     pub index: Url,
```

Comment:
```suggestion
    #[arg(long, value_name = "URL")]
    pub index: Option<Url>,
```

Wdyt about this approach? With this we can probably use package's source id to get registry url and thus avoid duplication.

Reply:
Yup, I've considered this but at the end opted to stick with clap's default value, for no particular reason. 
If you prefer it this way (and deduplication is always a valid argument) we can do that no prob. 
Changed in: https://github.com/software-mansion/scarb/pull/1570/commits/c6a5c0fd058c402b3fd37028513c9b33d9cf77fe

Reply:
Unresolving this because I needed to make some further changes. It appeared that source id taken from package is being wrong (it's a path). So in order to get the default index I did https://github.com/software-mansion/scarb/pull/1570/commits/ef6b9ad6ba85e264b44f52ccdab3dd2111c5e424